packages/terra-core-docs/src/terra-dev-site/doc/divider/About.1.doc.mdx

```diff
@@ -42,8 +42,8 @@ import Divider from 'terra-divider';
 
 ## Examples
 <DividerExample title="Default" />
-<DividerWithText title="Custom Text" />
-<DividerWithWrappingText title="Custom Text" />
+<DividerWithText title="Divider With Custom Heading Level" />
+<DividerWithWrappingText title="Divider With Custom Heading Level" />
```

Comment:
Let's remove this `DividerWithWrappingText` example as it does not illustrate best practices or recommended usage, and this case is already covered by tests to capture the wrapping text.